docs/sources/tempo/operations/cross_tenant_query.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Cross-tenant query federation
+menuTitle: Query
+description: Cross-tenant query federation
+weight: 70
+aliases:
+- /docs/tempo/operations/cross-tenant-query
+---
+
+
+# Cross-tenant query federation
+
+> NOTE: you need to enable `multitenancy_enabled: true` in the cluster for multi-tenant querying to work.
+> see [enable multi-tenancy]({{< relref "./multitenancy" >}}) for more details and implications of `multitenancy_enabled: true`.
+
+Tempo supports multi-tenant queries. where users can send list of tenants  multiple tenants.
+
+The tenant IDs involved need to be specified separated by a '|' character in the 'X-Scope-OrgID' header.
+
+cross-tenant query is enabled by default, and can be controlled using `multi_tenant_queries_enabled` config.
+
+```yaml
+query_frontend:
+   multi_tenant_queries_enabled: true
+```
+
+### Use cross-tenant query federation
+
+To submit a query across all tenants that your access policy has access rights to, you need to configure tempo datasource.
+
+Update Tempo datasource to send `X-Scope-OrgID` header with values of tenants separated by `|` e.g. `test|test1`, and query the tempo like you already do.
+
```

Comment:
Is this correct that we're updating a data source in Grafana? Also, data source should be two words. 
```suggestion
To submit a query across all tenants that your access policy has access rights to, you need to configure tempo data source in Grafana.

Update the Tempo data source to send `X-Scope-OrgID` header with values of tenants separated by `|`, for example, `test|test1`. Query the data source like you already do.

```

Reply:
I will drop this, we will add it back once we have more clarity on this.